src/WikibaseFacetedSearchExtension.php

```diff
@@ -374,4 +376,16 @@ public function newItemPageUpdater(): ItemPageUpdater {
 		);
 	}
 
+	public function newConfigEditPageTextBuilder( IContextSource $context ): ConfigEditPageTextBuilder {
+		return new ConfigEditPageTextBuilder(
```

Comment:
This name is weird now

Better to make it be about what it builds, rather than about where what it builds is used

Reply:
https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/220